steamed_hams.sh

```diff
@@ -39,9 +39,10 @@ elif [ "$2" = "debug" ] || [ "$2" = "engineeringdebug" ]; then
     start_exphan="sleep 0.001s; python3 experiment_handler/experiment_handler.py "$1" ; bash"
     start_radctrl="sleep 0.001s; python3 radar_control/radar_control.py; bash"
 #    start_datawrite="sleep 0.001s; python3 data_write/data_write.py --enable-bfiq --enable-pre-bfiq --enable-tx --enable-raw-rf; bash"
-    start_datawrite="sleep 0.001s; python3 data_write/data_write.py --enable-pre-bfiq --enable-raw-rf; bash"
+    start_datawrite="sleep 0.001s; python3 data_write/data_write.py --enable-pre-bfiq --enable-raw-rf --enable-raw-acfs; bash"
     start_usrp_driver="sleep 0.001s; source mode "$2" ; gdb -ex start usrp_driver 2>usrp_output.txt; bash"
-    start_dsp="sleep 0.001s; source mode "$2"; /usr/local/cuda/bin/cuda-gdb -ex start signal_processing; bash"
+#    start_dsp="sleep 0.001s; source mode "$2"; /usr/local/cuda/bin/cuda-gdb -ex start signal_processing; bash"
+    start_dsp="sleep 0.001s; source mode release; signal_processing; bash;"
```

Comment:
Why remove the command line argument for mode?
Also, should this be debug or engineeringdebug or does signal_processing not have that mode?

Reply:
should have multiple levels: 1) rawrf mode, 2) debug (fast w/ signal processing in release), 3) engineeringdebug (for all stages writing), 4) all modules debug (or something like so) ALSO 5) an output ptrs iq mode instead of having to edit the file all the time.

Reply:
change output_ptrs_iq to antennas_iq; change the prebfiq flag  to antennasiq flag

Reply:
@kkotyk

Reply:
Gonna improve the steamedhams stuff in another PR. doesnt affect this one.